packages/yoga/src/Snackbar/web/Snackbar.test.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import { ThemeProvider, Snackbar } from '../..';
+
+describe('<Snackbar />', () => {
+  it('should match snapshot', () => {
```

Comment:
I'd add some more tests here for:

* Hide icon property
* onAction and actionLabel
* onClose

Does it make sense?

Reply:
How about now